src/analysis/normed_space/star/basic.lean

```diff
@@ -193,3 +193,31 @@ variables {𝕜}
 lemma starₗᵢ_apply {x : E} : starₗᵢ 𝕜 x = star x := rfl
 
 end starₗᵢ
+
+section matrix
+
+local attribute [instance] matrix.normed_group
+
+open_locale matrix
+
+lemma matrix.entrywise_sup_norm_star_eq_norm {α : Type*} [normed_ring α] [star_add_monoid α]
+  [normed_star_monoid α] {n : Type*} [fintype n] :
+  ∀ (M : (matrix n n α)), ∥star M∥ = ∥M∥ :=
+begin
+  intro M,
```

Comment:
```suggestion
lemma matrix.entrywise_sup_norm_star_eq_norm {n : Type*} [normed_ring E] [star_add_monoid E]
  [normed_star_monoid E] [fintype n] (M : (matrix n n E)) : ∥star M∥ = ∥M∥ :=
begin
```
I think using `E` instead of `α` matches the rest of the file better (also it's already declared so you don't need to do it again -- if you want to check what is defined at a particular point in the code, you can use `#where`). Also, it's somewhat more usual to put arguments like `M` as an actual argument of the lemma instead of putting in a for-all statement, but this is purely esthetics, under the hood it's the exact same thing as far as I know.

Reply:
`#where` is a really nice tip, thanks!  I think that I ended up with `α` by trying to match style a different file, only to place it (correctly, but inconsistently) in this one.

I understand the preference for arguments over for-all statements.  I thought I needed to be consistent with what `normed_star_monoid` was asking for, but you're right, it takes this statement just fine.